### PR TITLE
Feature/stud 240/create teams page & Feature/stud 243/join team modal

### PR DESCRIPTION
**Pull request by yasfahm**
**STUD 240:**
- Created Teams page
- Changed tabbar icons

154856324-20268387-27d4-4c1a-b5c1-ab7c72800729

**STUD 243:**
- Create modal for joining a team

IMG_0465

### CODE REVIEW

File: src/components/modals/JoinTeam.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+const JoinTeamContainer = styled.View `
+  flex-direction: row;
+  flex-wrap: wrap;
+  background-color: #fff;
+  max-width: 60%;
+  justify-content: center;
+  align-items: center;
+  border-radius: 12;
+  padding-vertical: 10;
+  padding-horizontal: 10;
```
Comment (stellakimnguyen) [inline src/components/modals/JoinTeam.js]: Use theme.SPACING_SMALL for both paddings

File: src/components/modals/JoinTeam.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+`
+const TempButton = styled.Pressable `
+  background-color: #7A54F7;
+  align-items: center;
+  border-radius: 12;
+  padding-vertical: 10;
+  padding-horizontal: 10;
```
Comment (stellakimnguyen) [inline src/components/modals/JoinTeam.js]: Use theme.SPACING_SMALL for both paddings

File: src/components/modals/JoinTeam.js [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+  background-color: #fff;
+  max-width: 60%;
+  justify-content: center;
+  align-items: center;
+  border-radius: 12;
+  padding-vertical: 10;
+  padding-horizontal: 10;
+`
+const TempButton = styled.Pressable `
+  background-color: #7A54F7;
```
Comment (stellakimnguyen) [inline src/components/modals/JoinTeam.js]: Use theme.COLOR_PURPLE

File: src/components/modals/JoinTeam.js [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+  align-items: center;
+  border-radius: 12;
+  padding-vertical: 10;
+  padding-horizontal: 10;  
+`
+const TempButtonText = styled.Text `
+  color: #fff;
```
Comment (stellakimnguyen) [inline src/components/modals/JoinTeam.js]: Use theme.COLOR_WHITE

File: src/components/modals/JoinTeam.js [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+  margin-top: 6;
+`;
+
+const CancelButtonText = styled.Text `
+  align-items: center;
+  color: #fff;
+  font-weight: bold;
+  font-size: 12;
+  letter-spacing: 2;
+  margin-top: 6;
```
Comment (stellakimnguyen) [inline src/components/modals/JoinTeam.js:138]: For all number values that are common (to check in theme.style.js), use the theme.SPACING_MEDIUM, etc. It ensures that we have a cohesive design in the interface! From what I saw quickly, you can replace most letter-spacings, font-sizes, margins.

Comment (yasfahm) [reply src/components/modals/JoinTeam.js:138]: fixed it!

**Review by nouj — APPROVED** [review]

**Pull request merged**